dips/dip-10.md

```diff
@@ -11,49 +11,60 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+  * Privacy: There are concerns around leakage of PII when using subaddresses. Pay Address provides better privacy compared to sub-addresses, because unlike sub-addresses, Pay Address is intended to be used in tandem with an off-chain protocol. Hence, Pay Address does not need to expose PII to the chain 
+  * Persistent Identifiers: Currently there is a lack of persistent user identifiers in the Diem ecosystem. while subaddresses need to be refreshed for each payment, Pay Address is a persistent identifier from a user's perspective. From the perspective of the chain, Pay Addresss does not exist.
+
+# End-to-End Experience
+Below is an example of using Pay Address domain for transferring money from one user to another. 
+
+## Prerequisite:
+* VASPs get approval from association (via some offline process) on domain name
+```
+avasp, 0xf72589b71ff4f8d139674a3f7369c69b
+bvasp, 0xc5ab123458df0003415689adbb47326d
+```
 
-# User story
-Below is an example of using Diem ID domain for transferring money from one user to another. 
+## User Story: 
+* Bob wants to receive funds from Alice
+* Bob registers a Pay Address at VASP B: `bob@bvasp`
+* Alice's Pay Address is registered at VASP A: `alice@avasp`
+* Bob shares either this Pay Address, `bob@bvasp`, or a PaymentURI, `diem://bob@bvasp` with Alice
+* Alice logs into VASP A, enters Bob’s identifier, an amount to pay, and submits payment
+* Alice’s VASP (VASP A) contacts Bob’s VASP’s (VASP B) off-chain API with the sender identifier `alice@avasp` and requests a reference_id, `rb1`
+* VASP A constructs a transaction with the specified amount and the reference_id `rb` and submits it to the diem network
+* VASP B receives a transaction with metadata containing `rb1`, deposits the amount in his account, and attaches the relevant metadata (e.g., Alice’s Pay Address) to the transaction so Bob can confirm the transaction
 
-* Receiver shares their Diem ID with the sender (outside the scope of this document)
-* Sender enters receiver's Diem ID and other relevant information into their wallet application
-* Sender's wallet attempts to fetch the receiver's profile picture and full name along with a confirmation dialog
-* Sender confirms the identity of the recipient
-* Sender presses ‘send’ button
-* Receiver's account is credited the amount in the transaction
-* Receiver's wallet displays recent transactions and their senders
 
-# Diem ID format
-Diem ID is a string in format `<user-id>$<diem-id-domain>`.
+# Pay Address Format
+Diem defines globally unique identifiers to facilitate transacting on the Diem Payment Network. As Diem stores no PII on-chain, these identifiers are exchanged during the off-chain process between two VASPs to identify a distinct source and destinations within their respective VASPs in order to produce a reference_id that can be stored on-chain. The format for the identifiers follows:
```

Comment:
define PII ... might also be worth describing goals and properties before or after the user story

goals include:
* Eliminate the need for PII on-chain for peer-to-peer transactions
* Provide persistent, human-readable identifiers for sending and receiving transactions
any more?

Reply:
Are the goals listed in #Motivation section not enough? I can also just elaborate more there

Reply:
Hmm... this feels a little redundant, but whatever... :)